LiteCore/Database/DatabaseImpl.cc

```diff
@@ -51,7 +51,7 @@ namespace litecore {
 
 
     static constexpr slice kMaxRevTreeDepthKey = "maxRevTreeDepth";
-    static constexpr uint32_t kDefaultMaxRevTreeDepth = 20;
+    static constexpr uint32_t kDefaultMaxRevTreeDepth = 100;
```

Comment:
The max revTree depth is how many revs to store in the history of the revTree document, the Replicator's maxHistory is the max depth of history to send to SG during sync. We have a facility in `c4doc_getRevisionHistory` to generate random revIDs to fill in the gap between `maxRevTreeDepth` and Replicator's `maxHistory`, which seems like it was intended to be used based on the comments in the function, but never was because previously the values of `maxRevTreeDepth` and `maxHistory` were the same.

Reply:
In the latest commit, `maxRevTreeDepth`/`maxHistory` are different (`50`/`100`)